src/EDMF/Models/entr_detr.jl

```diff
@@ -7,6 +7,16 @@ struct BOverW2{FT} <: EntrDetrModel
   δ_factor::FT
 end
 
+struct RH_Diff{FT} <: EntrDetrModel
```

Comment:
```suggestion
struct RelativeHumidityDifference{FT} <: EntrDetrModel
```
Let's make this verbose, and follow Julia struct [naming conventions](https://climate-machine.github.io/CLIMA/latest/CodingConventions.html).